docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -19,8 +19,22 @@ the needs of the query.
 * **Type:** :ref:`prop-type-boolean`
 * **Default value:** ``true``
 
-Enable writer scaling. This can be specified on a per-query basis
-using the ``scale_writers`` session property.
+Enable writer scaling across different workers. In this, each worker will
+only get single writer task. So, the maximum scale it can reach is the
+total number of workers. This can be specified on a per-query basis using
+the ``scale_writers`` session property.
+
+``task.scale-writers``
+^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Enable writer scaling within a single worker task. The maximum scale it can reach
+is equal to the maximum of ``task.writer-count`` and ``task.min-scaled-writer-count``.
+Also, note that the default value of ``task.min-scaled-writer-count`` is 8, which
+means it'll scale up to 8 concurrent writers. This can be specified on a per-query
+basis using the ``task_scale_writers`` session property.
```

Comment:
```
Enable scaling the number of concurrent writers within a task. The maximum writer count per task
for scaling is the maximum of ``task.writer-count`` and ``task.min-scaled-writer-count``.
Additional writers are added only when the average amount of physical data written per writer is 
above the minimum threshold of ``writer-min-size`` and query is bottlenecked on writing.
This can be specified on a per-query basis using the ``task_scale_writers`` session property.
```